pkg/terminal/command.go

```diff
@@ -105,11 +104,16 @@ A tracepoint is a breakpoint that does not stop the execution of the program, in
 See also: "help on", "help cond" and "help clear"`},
 		{aliases: []string{"restart", "r"}, cmdFn: restart, helpMsg: `Restart process.
 
-  restart [-c checkpoint] [-args newargv...]
+  restart
```

Comment:
cc @derekparker 

Personally I think this would be better like this:

```
       restart [-args newargv...]
       restart [checkpoint]

For recorded processes restarts from the start or from the specified checkpoint.
For normal processes restarts the process, optionally changing the arguments.
```